Modules/VMware.Hv.Helper/VMware.HV.Helper.psm1

```diff
@@ -2140,6 +2140,15 @@ function New-HVFarm {
 .PARAMETER Spec
     Path of the JSON specification file.
 
+.PARAMETER NumCPU
+    Number of CPU of the Vm Instances
+
+.PARAMETER Ram
```

Comment:
I think a better name for this parameter would be something like MemoryMB, which shows in what units the parameter should be specified. Or if you don't like that at least add this information in the parameter description.